README.md

```diff
@@ -641,4 +641,63 @@ Feature: Gestionar los habilidades para los usuarios
         Given el servicio de gestión de habilidades "skill-controller" sin autenticación válida 
```

Comment:
Add in Table of content the step 7 you're creating
![image](https://user-images.githubusercontent.com/36433884/201161501-35da166a-8a38-4ecb-8bd1-19582b692538.png)

Reply:
Done